### PR TITLE
Aa mk/admin tool project (local run only for now)

### PR DESCRIPTION
**Pull request by mkuzdowicz**
## What does this change?
Introducing new admin tools project for re-ingestion and image projection

This PR introduce only image projection
and have local run only for now

## How can success be measured?
run the project locally by `./dev-start.sh` 
access 

https://admin-tools.media.local.dev-gutools.co.uk/images/{id}/project

to see the image projection

## Tested?
- [x] locally
- [ ] on TEST

### CODE REVIEW

**Comment by mkuzdowicz** [comment]
> This looks good, but I probably can't approve as I worked on it.
> 
> One question about using env-vars; are we happy to pass secrets (the api key) like this?

I added this like that because i was thinking that lambda will take that as env vars, but we can use secrets manager instead

- that will change a lot anyway as this PR is only for local run
and next work will bring restrcuture

**Comment by mkuzdowicz** [comment]
> This looks good, but I probably can't approve as I worked on it.
> 
> One question about using envvars; are we happy to pass secrets (the api key) like this?

i commented that already before

But i guess as we use play only for local run i am happy to even hardcode it in this PR as for local run it does not matter

and we can get rid of env vars for local run

**Comment by mkuzdowicz** [comment]
> > This looks good, but I probably can't approve as I worked on it.
> > One question about using envvars; are we happy to pass secrets (the api key) like this?
> 
> i commented that already before
> 
> But i guess as we use play only for local run i am happy to even hardcode it in this PR as for local run it does not matter
> 
> and we can get rid of env vars for local run

env vars removed

File: dev-start.sh [hunk 1]
```diff
@@ -115,6 +115,7 @@ setupLocalKinesis() {
   aws --profile media-service --region=eu-west-1 --endpoint-url=http://localhost:4568 kinesis list-streams
 }
 
+
```
Comment (aug24) [inline dev-start.sh]: Remove redundant changed whitespace?

**Pull request merged**